apollo-backend/src/test/java/io/logz/apollo/helpers/ModelsGenerator.java

```diff
@@ -69,6 +69,13 @@ public static Environment createAndSubmitEnvironment(ApolloTestClient apolloTest
         return testEnvironment;
     }
 
+    public static Environment createAndSubmitEnvironment(ApolloTestClient apolloTestClient, int concurrencyLimit) throws ApolloClientException {
+        Environment testEnvironment = ModelsGenerator.createEnvironment();
+        testEnvironment.setConcurrencyLimit(concurrencyLimit);
```

Comment:
Why here? put it in the models generator, or have a very specific method

Reply:
This is already in the ModelsGenerator

Reply:
in createEnvironment i meant